rock-gazebo/simulation-gazebo_thruster
# Include thruster disturbances

Add disturbances sampled from gaussian distributions for each thruster effort in order to emulate input disturbances to the system. 

Test procedure:
* Checkout this branch
* Configure each thruster disturbance parameters (mu, sigma) in the sdf file 
* Build the solution and run the simulation 
* Use the move_to_position or other action to make the robot move
* Inspect on plot2D the thruster efforts output ports in the gazebo_thruster element
* Check whether the robot was able to achieve the goal
